src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -631,12 +626,7 @@ private PSObject GetSyntaxObject(CommandInfo command)
                         }
                         else
                         {
-                            replacedSyntax = string.Format(
-                                "{0} (alias) -> {1}{2}{3}",
-                                aliasName,
-                                command.Name,
-                                Environment.NewLine,
-                                command.Syntax.Replace(command.Name, aliasName));
+                            replacedSyntax = string.Format($"{aliasName} (alias) -> {command.Name}{Environment.NewLine}{command.Syntax.Replace(command.Name, aliasName)}");
```

Comment:
string.Format ?

Reply:
String.Create doesn't accept only 1 argument, what format provider should I add?

Reply:
Yes, we can add provider. I think Invariant is ok since it is strings only.